site/en/blog/chrome-120-beta-whats-new-for-extensions/index.md

```diff
@@ -0,0 +1,113 @@
+layout: 'layouts/blog-post.njk'
+title: What's new in Chrome 120 for Extensions
+description: >
+  A summary of important changes in Chrome 120 for Chrome Extension developers.
+subhead: >
+  Chrome 120 is now available in beta and includes many exciting updates for Chrome Extension developers. 
+date: 2023-11-02
+hero: 'image/6hHqS5auVgWhN0cQNQztaJx5w4M2/yNTK9kjwbKdM0ZFcqnui.png'
+alt: 'Chrome 120 Beta'
+authors:
+  - sebastianbenz
+tags:
+  - extensions-news
+---
+
+It’s been a busy year for the Chrome Extensions team. With yesterday’s [Chrome 120 Beta release](/blog/chrome-120-beta/), the extensions platform is making another big step forward.  For an overview on what’s happened this year, check out our quarterly updates from 
+[July](/blog/extension-news-july-2023/) and [October](/blog/extension-news-october-2023/). Read on for what's new in Chrome 120 for Extensions.
+
+## Closing the platform gap
+
+With the release of Chrome 120, we will close the remaining platform gaps listed on our [Manifest V3 known issues page](/docs/extensions/migrating/known-issues/). The new userScript API  as well as support for fileHandler on ChromeOS have been the two remaining items on the list that we can now cross off, Together with the changes described in the previous quarterly update we are really happy about the current state of the Chrome extension platform and what we’ve accomplished over the past year.
+
+## New userScripts API
+
+User script support has landed! User scripts are (usually relatively small) snippets of code that extensions can inject into web pages in order to modify the page's appearance or behavior. They can be created directly by the user or discovered in a number of different user script repositories around the web. Starting with Chrome 120 Manifest V3 extensions can now manage the collection of user scripts and determine when and how to inject them on web pages. 
+
+There is one significant difference between user script support in Manifest V2 and Manifest V3. As user scripts are powerful and require high trust in the author of the user script, the Chrome team decided that users must opt into Developer mode before they can run a user script. 
+
+{% Img src="image/6hHqS5auVgWhN0cQNQztaJx5w4M2/ktS60KiJNxwjd3PSIEkb.png", alt="Screenshot showing the extensions management tab", width="800", height="164", class="screenshot screenshot--filled" %}
+
+
+
+
+Our new [userScript sample](LINK) demonstrates a simple approach for detecting whether Developer mode is enabled and providing a simple onboarding flow.
+
+
+
+<figure>
+{% Img src="image/6hHqS5auVgWhN0cQNQztaJx5w4M2/Fh4eS4ovF8QboLiwLhY4.png", alt="Sample onboarding flow for user scripts.", width="800", height="401" %}
+<figcaption>Onboard users when Developer mode is disabled</figcaption>
+</figure>
+
+To get started check out the [documentation](LINK) or take a look at the [official sample](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/api-samples/userScripts).
```

Comment:
```suggestion
To get started check out the [documentation](/docs/extensions/reference/userScripts/) or take a look at the [official sample](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/api-samples/userScripts).
```

Reply:
You should double check this link since it's not live yet.

Reply:
I checked on the preview